src/common/header/Header.tsx

```diff
@@ -102,8 +115,26 @@ function Header(): React.ReactElement {
       >
         <HDSHeader.LanguageSelector sortLanguageOptions={sortLanguageOptions} />
 
-        <hr aria-hidden="true" />
-        <UserDropdown />
+        {/* TODO: Translate text*/}
```

Comment:
I'm guessing this TODO can be now removed 🤔

Reply:
Removed.